.github/workflows/version_check.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check Semantic Versioning
+on:
+  - pull_request
+
+jobs:
+  version-check:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Check version
+        id: check-version
+        run: |
+          if [[ $(git show main:VERSION &> /dev/null) -eq 0 ]] &&  # Ignore if file doesn't exist yet
+             [[ $(git diff origin/main origin/${GITHUB_HEAD_REF} -- VERSION | wc -c) -eq 0 ]]
```

Comment:
This one works as expected because we're piping to `wc -c`. If `wc -c` is 0, that means there are no changes between the files. If this check wasn't working correctly, this action would be failing on this PR with 'VERSION file must be updated'.